Add somre more folders to eslint import order rules

lfryemason opened the pull request:
In order to improve code cleanliness slightly, we are rolling out an eslint rule to the entire repo that ensure .js, .jsx, .ts and .tsx files have imports automatically sorted. 

We are rolling this out incrementally to prevent errors and decrease PR size. This PR removes a subset of folders from the rule exclusions.

FLUP of [#57247](https://github.com/code-dot-org/code-dot-org/pull/57247)

**For PR reviewers**: please double check that all comments in imports are appropriately moved with the line they are talking about.

## Links

<!--
  Links to relevant external resources; ie, specification documents, Jira tickets, related PRs, Honeybadger errors, etc.
-->

<!--
- spec: []()
- jira ticket: []()
-->

## Testing story

<!--
  Does your change include appropriate tests?
  If so, please describe how the tests included in this PR are sufficient.
  If not, please explain why this change does not need to be tested.
-->

<!-- Other aspects to consider. Delete any sections that are not relevant to your change. -->

## Deployment strategy

## Follow-up work

<!--
  List (ideally with Jira links) any clean-up or technical debt that will be addressed in future work.
-->

## Privacy

<!--
  1.	Does this change involve the collection, use, or sharing of new Personal Data?
  2.	Does this change involve a new or changed use or sharing of existing Personal Data?
-->

## Security

<!-- Link to Jira task(s) where sensitive security issues are discussed privately. -->

## Caching

## PR Checklist:

<!--
  The final step! Before you create your PR, double-check that everything is in order.
  Change [ ] to [X] during creation to check boxes.
-->

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

Hamms reviewed (commented):
Do the circular dependency errors you were seeing represent actual new circular dependencies, or are they just the result of the existing circular dependencies showing up in a different order? See https://github.com/code-dot-org/code-dot-org/pull/58887#issuecomment-2135327537 for an example

lfryemason commented:
> Do the circular dependency errors you were seeing represent actual new circular dependencies, or are they just the result of the existing circular dependencies showing up in a different order? See [#58887 (comment)](https://github.com/code-dot-org/code-dot-org/pull/58887#issuecomment-2135327537) for an example

They are because the order of imports change. I assume this is because the existing circular dependencies don't cause problems when they are imported in a specific order, but this would take time to confirm.

Hamms commented:
I think it's worth taking that time! Updating the circular dependencies tracker at `apps/circular_dependencies.json` should be pretty straightforward

lfryemason commented:
> circular_dependencies

I didn't know about this `circular_dependencies.json` list. I removed the other places I disabled the import rule and also added some clarifying text to the output so it is easier to find for the next person.

Hamms commented on apps/webpack.config.js:
```diff
@@ -109,7 +109,7 @@ const nodePolyfillConfig = {
           compilation.errors.push(
             new Error(
-              `Circular Dependency Checker : New Circular Dependency found : ${pathString}`
+              `Circular Dependency Checker : A new Circular Dependency found.\nKnown circular dependencies can be found in 'apps/circular_dependencies.json'\n Circular dependency: ${pathString}.`
```
:+1:

Hamms reviewed (approved):
Yeah, this circular dependencies functionality is relatively new. Thanks for taking the time to make it more discoverable!

This changeset LGTM, once that merge conflict gets resolved. Thank you!
